boards/realtek/rts5912_evb/rts5912_evb.dts

```diff
@@ -41,6 +41,7 @@
 	status = "okay";
 	pinctrl-0 = <&uart_rx_gpio113 &uart_tx_gpio114>;
 	pinctrl-names = "default";
+	gpios = <&gpioh 1 0>;
```

Comment:
how does one go from, say `gpio113` to `gpioh1`? anyway this should probably be called `rx-gpios` or something more descriptive. The for the last zero it's common practice to use `PWM_POLARITY_NORMAL` instead.